### PR TITLE
feat(node-sdk): Initialize only once

### PR DESCRIPTION
**Pull request by roncohen**

### CODE REVIEW

File: packages/node-sdk/src/utils.ts [hunk 1]
```diff
@@ -160,3 +160,16 @@ export function hashObject(obj: Record<string, any>): string {
 
   return hash.digest("hex");
 }
+
+export function once<T extends () => ReturnType<T>>(
```
Comment (pavkam) [inline packages/node-sdk/src/utils.ts:164]: Nice to have a test to validate all cases

Comment (roncohen) [reply packages/node-sdk/src/utils.ts:164]: do you mean, it would be nice to have? Aka, please add a test?

Comment (pavkam) [reply packages/node-sdk/src/utils.ts:164]: yes :)

Comment (roncohen) [reply packages/node-sdk/src/utils.ts:164]: don!

**Review by pavkam — APPROVED** [review]

**Pull request merged**